app/renderer/components/common/dropdown.js

```diff
@@ -67,13 +75,21 @@ const styles = StyleSheet.create({
       outlineWidth: '1px'
     }
   },
+
   commonForm: {
     backgroundColor: '#fff',
     fontSize: globalStyles.fontSize.flyoutDialog
   },
+
   settings: {
     width: '280px'
   },
+
+  settings_panel: {
+    // ref: browserButton_panelItem
+    width: '180px'
```

Comment:
if `browserButton_panelItem` shares the same value then it should be both hosted in `global.js`, then the comment can be removed